src/core/documentBuilder.ts

```
@@ -87,11 +105,15 @@ export interface SignOptions {
  */
 export class DocumentBuilder {
   private document: Partial<VerifiableCredential>; // Holds the document to be built and signed.
-  private documentType: string = 'w3c'; // Default to W3C
-  private selectedStatusType: 'transferableRecords' | 'verifiableDocument' | null = null; // Tracks selected status type.
+  private readonly documentType: string = 'w3c'; // Default to W3C
+  private selectedStatusType:
+    | 'transferableRecords'
+    | 'obligationRecords'
+    | 'verifiableDocument'
+    | null = null; // Tracks selected status type.
```

ESLint

[error] Replace `|·'transferableRecords'⏎····|·'obligationRecords'⏎····|·'verifiableDocument'⏎···` with `'transferableRecords'·|·'obligationRecords'·|·'verifiableDocument'`

(prettier/prettier)